pkg/status/status.go

```diff
@@ -329,7 +334,7 @@ func (s *status) monitorClusterStatus() {
 			if cohelpers.IsStatusConditionFalse(s.clusterOperator.Status.Conditions, configv1.OperatorAvailable) {
 				conditionListBuilder := clusterStatusListBuilder()
 				conditionListBuilder(configv1.OperatorProgressing, configv1.ConditionFalse, "")
-				statusConditions := conditionListBuilder(configv1.OperatorAvailable, configv1.ConditionTrue, fmt.Sprintf("%s is available", s.version))
+				statusConditions := conditionListBuilder(configv1.OperatorAvailable, configv1.ConditionTrue, fmt.Sprintf("%s is available", s.releaseVersion))
```

Comment:
Looks good to me. 

Just a `nit`: Currently the message would print as `<RELEASE_VERSION unavailable> is available`. Can we think of better phrasing for the message? Something like `Release version: <Unknown RELEASE_VERSION> is available`. In the ideal situation the message would read `Release version: 2.0.5 is available`.

Reply:
I support this change.

Reply:
Or we can use the github commit hash and show that as the release version in the case the environment variable is not present. OLM does this or at lease used to do this in the past.

Reply:
I am not confident that we should set the release version to the marketplace commit hash.

The `RELEASE_VERSION` environment variable should always be set when on a cluster managed with CVO. It represents the version of the cluster and by extent the version of the operator. If the `RELEASE_VARIABLE` environment variable was not set the install/upgrade would fail.

This bug should only occur when the marketplace operator is deployed manually. I believe that having the release version set to the marketplace sha could misrepresent it's intended purpose.

If we want to provide users with a way to identify the version marketplace apart from the CVO Release Version, I believe we should create a new task where we add the commit has of the marketplace operator as a separate key/value pair within versions.

Reply:
@aravindhp ^

Reply:
This will help if someone is using marketplace in upstream kubernetes cluster and not just from the CLI. I also think it would be immensely useful from a CLI perspective.

Reply:
@aravindhp could you update this comment based on the off line conversation that we had.

Reply:
This should state something to the effect that we are running the operator that is not part of the payload. For upstream we should use the git has to show the version elsewhere as part of an another PR.

Reply:
@aravindhp
>This should state something to the effect that we are running the operator that is not part of the payload.

Thanks!

> For upstream we should use the git has to show the version elsewhere as part of an another PR.

The status routine won't start on upstream.

Reply:
@kevinrizza and I fell on setting the `ClusterOperator` version to `OpenShift Independent Version` if `RELEASE_VERSION` is not set.

Reply:
>The status routine won't start on upstream.

My statement still stands: " For upstream we should use the git hash to show the version **elsewhere** as part of another PR."

Reply:
Ah - missed that elsewhere, thank you.